channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -201,6 +205,10 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControlsAI {
             };
 
             _self.teardown = _notImplemented;
+
+            _self.addHeader = (name: string, value: string) => {
```

Comment:
@MSNev Does it make sense to add endpointUrl check on this method? With current implementation, even the endpoinUrl is Breeze, user can call `sender.addHeader` to add custom headers and fails track.

Reply:
No, I don't think so... If the request fails because they add (effectively) invalid headers for breeze then this would be considered a programming error (I think). Most people won't need to or use this..